config/v1/types_cluster_version.go

```diff
@@ -88,6 +88,21 @@ type ClusterVersionSpec struct {
 	// +optional
 	Capabilities *ClusterVersionCapabilitiesSpec `json:"capabilities,omitempty"`
 
+	// SignatureStores contains the upstream URIs to verify release signatures.
+	// By default, CVO will use existing signature stores if this property is empty.
+	// The CVO will check the release signatures in the local ConfigMaps first. It will search for a valid signature
```

Comment:
Which local configmaps, are there multiple configmaps? How are these configmaps selected?

Reply:
Docs for the ConfigMap signature store are [here][1], but in practice this is not user-facing.  `oc adm release mirror ...` ([source][2]) and `oc-mirror ...` ([source][3]) pack the signatures into ConfigMap manifests on behalf of the user, who can apply them to their target cluster without having to look inside.

[1]: https://github.com/openshift/library-go/blob/3ff870451f43cb45e7566dd7e9d8d43040b1248a/pkg/verify/store/configmap/configmap.go#L25-L33
[2]: https://github.com/openshift/oc/blob/3b3f931a46f44b4e6fdf8630cb45dbbfc108239e/pkg/cli/admin/release/mirror.go#L357
[3]: https://github.com/openshift/oc-mirror/blob/8f9caefdc1e0255141f8bf282f473733f7b16e55/pkg/cli/mirror/release.go#L446